app.json

```diff
@@ -140,8 +140,8 @@
     "MITXPRO_ENVIRONMENT": {
       "description": "The execution environment that the app is in (e.g. dev, staging, prod)"
     },
-    "MITXPRO_FROM_EMAIL": {
-      "description": "E-mail to use for the from field"
```

Comment:
Is `MITXPRO_FROM_EMAIL` not in use anywhere?

Reply:
No it isn't, we have `MAILGUN_FROM_EMAIL` for using as `form` address.